lib/lotus/generators/model/entity_spec.minitest.tt

```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
```

Comment:
Since an entity spec should be able to be run without loading the entire application code there should be an `relative_require` for the entity. At least that's what I'm thinking :-)

[Here](https://github.com/lotus/lotus/blob/master/lib/lotus/generators/action/action_spec.minitest.tt#L2) you can see how it is done for the action spec generator.